cluster/images/etcd/migrate/BUILD

```diff
@@ -4,19 +4,31 @@ load(
     "@io_bazel_rules_go//go:def.bzl",
     "go_binary",
     "go_library",
+    "go_test",
 )
 
 go_binary(
-    name = "rollback",
+    name = "migrate",
     embed = [":go_default_library"],
 )
 
 go_library(
     name = "go_default_library",
-    srcs = ["rollback.go"],
-    importpath = "k8s.io/kubernetes/cluster/images/etcd/rollback",
+    srcs = [
+        "data_dir.go",
+        "migrate.go",
+        "migrate_client.go",
+        "migrate_server.go",
+        "migrator.go",
+        "rollback_v2.go",
+        "versions.go",
+    ],
+    importpath = "k8s.io/kubernetes/cluster/images/etcd/migrate",
     deps = [
         "//third_party/forked/etcd221/wal:go_default_library",
+        "//vendor/github.com/blang/semver:go_default_library",
```

Comment:
We now have two semver libraries and I am not sure they agree on what is a valid version number.